upbound.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: meta.dev.upbound.io/v1alpha1
+kind: Project
+metadata:
+  name: configuration-azure-network
+spec:
+  dependsOn:
+  - provider: xpkg.upbound.io/upbound/provider-azure-network
+    version: v1
+  - function: xpkg.upbound.io/crossplane-contrib/function-auto-ready
+    version: '>=v0.0.0'
+  description: This is where you can describe your project.
+  license: Apache-2.0
+  maintainer: Upbound User <user@example.com>
+  readme: |
```

Comment:
Update to the readme from the good old crossplane.yaml ?

Reply:
Done